PYTHIA6/AliPythia6/AliGenPythiaPlus.cxx

```diff
@@ -1381,81 +1339,365 @@ void AliGenPythiaPlus::GetSubEventTime()
   return;
 }
 
+///
+/// \return true if particle in central barrel acceptance
+///
+/// \param eta        particle pseudorapidity,  etamin=-etamax
+///
+Bool_t AliGenPythiaPlus::IsInBarrel(Float_t eta) const
+{
+  if( eta < fTriggerEta  ) 
+  return kTRUE;
+  else 
+  return kFALSE;
+}
```

Comment:
I thought of doing this, but preferred simplicity and there are subtleties between the 2 generators that cannot be treated similarly (status code only 0 or 1 in pythia8, label index+1 in pythia8). I will do the requested changes